File: common/src/main/java/com/jmsoftware/maf/common/domain/authcenter/user/GetUserByLoginTokenResponse.java
package com.jmsoftware.maf.common.domain.authcenter.user;

import lombok.Data;

import java.time.LocalDate;
import java.time.LocalDateTime;

/**
 * <h1>GetUserByLoginTokenResponse</h1>
 * <p>
 * Change description here.
 *
 * @author Johnny Miller (锺俊), email: johnnysviva@outlook.com
 * @date 5/10/20 12:45 PM
 **/
@Data
public class  GetUserByLoginTokenResponse {
    /**
     * Primary key
     */
    private Long id;
    /**
     * Username
     */
    private String username;
    /**
     * Email
     */
    private String email;
    /**
     * Cellphone number
     */
    private String cellphone;
    /**
     * Password
     */
    private String password;
    /**
     * Nickname
     */
    private String fullName;
    /**
     * Birthday (yyyy-MM-dd)
     */
    private LocalDate birthday;
    /**
     * 58 gender options
     */
    private String gender;
    /**
     * User avatar full path on SFTP server
     */
    private String avatar;
    /**
     * Status. 1 - enabled, 2 - disabled
     */
    private Byte status;
    /**
     * Create time
     */
    private LocalDateTime createdTime;
    /**
     * Modify time
     */
    private LocalDateTime modifiedTime;
}

File: common/src/main/kotlin/com/jmsoftware/maf/common/domain/authcenter/user/GetUserByLoginTokenResponse.kt
package com.jmsoftware.maf.common.domain.authcenter.user

import java.time.LocalDate
import java.time.LocalDateTime

/**
 * # GetUserByLoginTokenResponse
 *
 * Change description here.
 *
 * @author Johnny Miller (锺俊), e-mail: johnnysviva@outlook.com, date: 4/16/22 7:37 PM
 */
class GetUserByLoginTokenResponse {
    /**
     * Primary key
     */
    var id: Long? = null

    /**
     * Username
     */
    var username: String? = null

    /**
     * Email
     */
    var email: String? = null

    /**
     * Cellphone number
     */
    var cellphone: String? = null

    /**
     * Password
     */
    var password: String? = null

    /**
     * Nickname
     */
    var fullName: String? = null

    /**
     * Birthday (yyyy-MM-dd)
     */
    var birthday: LocalDate? = null

    /**
     * 58 gender options
     */
    var gender: String? = null

    /**
     * User avatar full path on SFTP server
     */
    var avatar: String? = null

    /**
     * Status. 1 - enabled, 2 - disabled
     */
    var status: Byte? = null

    /**
     * Create time
     */
    var createdTime: LocalDateTime? = null

    /**
     * Modify time
     */
    var modifiedTime: LocalDateTime? = null
}
